products/healthcare/terraform.yaml

```diff
@@ -89,14 +89,27 @@ overrides: !ruby/object:Overrides::ResourceOverrides
       - !ruby/object:Provider::Terraform::Examples
         name: "healthcare_hl7_v2_store_basic"
         skip_test: true
-        primary_resource_id: "default"
+        primary_resource_id: "store"
         vars:
           dataset_name: "example-dataset"
           hl7_v2_store_name: "example-hl7-v2-store"
           pubsub_topic: "hl7-v2-notifications"
+      - !ruby/object:Provider::Terraform::Examples
+        name: "healthcare_hl7_v2_store_parser_config"
+        skip_test: true
```

Comment:
I'm not sure why the tests were skipped originally, but it's probably worth un-skipping them to see if they work! You may have to put `min_version: beta` in here to do so. If they do, then there might be some cleanup of the handwritten tests that you could do (haven't actually really checked though)